systers_portal/meetup/forms.py

```diff
@@ -240,3 +259,21 @@ class Meta:
         helper_class = SubmitCancelFormHelper
         helper_cancel_href = "{% url 'view_support_request' meetup.slug" \
                              " support_request.pk %}"
+
+
+class PastMeetup(ModelFormWithHelper):
+    images = MultiFileField(required=False)
+
+    class Meta:
+        model = Meetup
+        fields = ('resources',)
+        helper_class = SubmitCancelFormHelper
+        helper_cancel_href = "{% url 'view_meetup' meetup.slug %}"
+
+    def save(self, commit=True):
+        """Override save to add created_by and meetup_location to the instance"""
+        instance = super(PastMeetup, self).save(commit)
+        if self.cleaned_data['images']:
+            for img in self.cleaned_data['images']:
+                MeetupImages.objects.create(image=img, meetup=instance)
+        return instance
```

Comment:
@ritwickraj78 Should be PastMeetupForm. Also, as we already have an EditMeetupForm where we can edit the image field what is the need for this form?